sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/sweep/parameterized_sweep.py

```diff
@@ -137,6 +142,21 @@ def set_limits(
             if trial_timeout is not None:
                 self.limits.trial_timeout = trial_timeout
 
+    def set_queue_settings(self, *, job_tier: Optional[str] = None, priority: Optional[str] = None):
+        """Set QueueSettings for the job.
+        :param job_tier: determines the job tier.
+        :type job_tier: str
+        :param priority: controls the priority on the compute.
+        :type priority: str
+        """
+        if self.queue_settings is None:
+            self.queue_settings = QueueSettings
```

Comment:
Is this intentional ? assigning class to class member

Reply:
Was a minute late in pushing out, fixed now